src/@next/Menu/components/MenuOptionLabel.tsx

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
 import { Typography } from '../../Typography';
 import { Neutral } from '../../utilities/colors';
+import { MenuOptionLabelContainer } from './MenuOptionLabelStyle';
+
+export const MenuOptionLabel = ({
+  label,
+  sublabel,
+}: {
+  label: React.ReactNode;
+  sublabel?: React.ReactNode;
+}) => {
+  if (!sublabel) {
+    return (
+      <Typography as="span" variant="body1" color={Neutral.B18}>
+        {label}
+      </Typography>
+    );
+  }
 
-export const MenuOptionLabel = ({ label }: { label: React.ReactNode }) => {
   return (
-    <Typography as="span" variant="body1" color={Neutral.B18}>
-      {label}
-    </Typography>
+    <MenuOptionLabelContainer sublabel={sublabel}>
+      <Typography as="span" variant="body1" color={Neutral.B18}>
+        {label}
+      </Typography>
+      {sublabel && (
```

Comment:
This && condition is unnecessary, we already have an early return, subLabel should be defined at this point

Reply:
Sorry I missed it. Updated!